content/posts/May-2022.md

```diff
@@ -31,3 +31,13 @@ weight: 62
 - When initializing source-git repos, the author of downstream commits created from patch files which are not in a git-am format is set to the original author of the patch-file in dist-git, instead of using the locally configured Git author. ([packit#1575](https://github.com/packit/packit/pull/1575))
 - Packit now correctly inform users about downstream errors only on the last try. (Previously, Packit informed for all tries even the last try succeded.) ([packit-service#1485](https://github.com/packit/packit-service/pull/1485))
 - Dashboard now also shows propose-downstream jobs in pipelines view. We have also merged all jobs to one column to minimize dead space on the page. ([dashboard#178](https://github.com/packit/dashboard/pull/178))
+
+## Week 19 (May 10th - May 16th)
+
+- Metadata dictionary is no longer required when specifying a job. Keys which used to belong to the yaml metadata dictionary are now keys of the job dictionary itself. ([packit#1569](https://github.com/packit/packit/pull/1569))
+- Packit now correctly removes patches during `packit source-git init` when the preamble does not contain blank lines. ([packit#1582](https://github.com/packit/packit/pull/1582))
+- Packit now supports `--release-suffix` parameter in all of the related CLI commands. Also we have added a support for the `release_suffix` option from configuration to the CLI. With regards to that we have introduced a new CLI switch `--default-release-suffix` that allows you to override the configuration option to Packit-generated default option that ensures correct NVR ordering of the RPMs. ([packit#1586](https://github.com/packit/packit/pull/1586))
+- Resolved an SRPM build problem caused by a new version of git that refuses to fetch in a git repo when it's owned on the OS level by someone else. ([packit#1497](https://github.com/packit/packit-service/pull/1497))
+- Packit now passes `PACKIT_COPR_PROJECT` and `PACKIT_COPR_RPMS` variables to the Testing Farm. `PACKIT_COPR_PROJECT` holds Copr project in format owner/project and `PACKIT_COPR_RPMS` space-separated RPMs that were built in Copr. ([packit-service#1486](https://github.com/packit/packit-service/pull/1486))
+- Packit now builds only its own dist-git commits. Other commits are not being acted upon. For reasoning, see [packit-service#1490](https://github.com/packit/packit-service/issues/1490). ([packit-service#1498](https://github.com/packit/packit-service/pull/1498))
+- We have automated our allowlisting process via a new Packit comment command `/packit verify-fas`. You can find more info in [our requirements](https://packit.dev/docs/packit-service/#requirements-for-running-packit-service-jobs). ([packit-service#1487](https://github.com/packit/packit-service/pull/1487))
```

Comment:
@majamassarini @mfocko @TomasTomecek @lbarcziova  Just wow :open_mouth: Really impressive for a single week! :rocket: